src/main/java/frc/robot/subsystems/VisionControl.java

```diff
@@ -166,7 +166,7 @@ public void teleopPeriodic() {
         }
     }
 
-    public void followPath() { //FIXME: Move this to chassis
+    public void followPath() {
```

Comment:
This is fine here because this is where the arrays are stored

Reply:
Yes. We are headed away from the encoder replay method for auto and toward using vision, so we might eventually remove this, but I agree it belongs here for now.

Reply:
👍